### PR TITLE
Master fix

### PR DESCRIPTION
**Pull request by aturkewi**
@Lukeghenco Can you take a look at this?

The biggest change was had to change a test from `calledOnce` is just `called`. For some reason, the test was registering as it being called twice.

### CODE REVIEW

File: components/SearchField.js [hunk 1]
```diff
@@ -2,8 +2,12 @@
 
 const React = require('react');
 
-const SearchField = props => (
-  <input />
-);
+class SearchField extends React.Component {
```
Comment (Lukeghenco) [inline components/SearchField.js]: const is more accurate here as this should be a `pure/stateless` component. You only create Classes if they hold state.

File: components/SearchResults.js [hunk 2]
```diff
@@ -2,8 +2,14 @@
 
 const React = require('react');
 
-const SearchResults = ({ results }) => (
-  <ul />
-);
+class SearchResults extends React.Component {
```
Comment (Lukeghenco) [inline components/SearchResults.js]: const is more accurate here as this should be a pure/stateless component. You only create Classes if they hold state.

File: test/Autocomplete-test.js [hunk 3]
```diff
@@ -68,7 +68,7 @@ describe('<Autocomplete />', function() {
     expect(wrapper.type()).toBe('div');
   });
 
-  it('should have class .search-field', function() {
+  it('should have class .autocomplete', function() {
```
Comment (Lukeghenco) [inline test/Autocomplete-test.js:71]: haha, for sure. These seem like unneeded tests though.

File: test/SearchResults-test.js [hunk 4]
```diff
@@ -7,7 +7,7 @@ const sinon = require('sinon');
 const SearchResults = require('../components/SearchResults');
 
 describe('<SearchResults />', function() {
-  it('should render an <ul />', function() {
+  it('should render an <ul>', function() {
```
Comment (Lukeghenco) [inline test/SearchResults-test.js:10]: LGTM 👍  If this was a component though I would add those self closing tags. Example `<AutoComplete />`

File: README.md [hunk 5]
```diff
@@ -12,87 +12,74 @@ Search API.
 
 This is what it's going to look like:
 
-![Screenshot](./assets/screenshot.png)
```
Comment (Lukeghenco) [inline README.md:15]: All of this text stuff looks good

**Review by Lukeghenco — CHANGES_REQUESTED** [review]
Thanks Avidor. Change back to pure components though.

**Comment by aturkewi** [comment]
Made the changes. Thanks @Lukeghenco

**Pull request merged**